<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="space" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>881</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="menu" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QPushButton" name="plusMenu">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>SimSun-ExtB</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius: 20px;
border: 1px solid rgb(153, 185, 255);
font: 36pt &quot;SimSun-ExtB&quot;;
background-color : rgb(255,255,255);
}
QPushButton:hover
{
color: rgb(1, 24, 156);
font: 36pt &quot;SimSun-ExtB&quot;;
border: 1px solid rgb(222, 233, 255);
background-color: rgb(222, 233, 255);
border-radius: 20px;
}
QPushButton:pressed
{
color: rgb(0,0,0);
font: 36pt &quot;SimSun-ExtB&quot;;
border-radius: 20px;
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="iconSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="Menu" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="fileUpload">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>26</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
color: rgb(0, 0, 0);
font: 10pt &quot;Nunito Medium&quot;;
border : 1px solid rgb(222, 233, 255);
background-color: rgb(255, 255, 255);
}
QPushButton:hover
{
color: rgb(1, 24, 156);
font: 10pt &quot;Nunito Black&quot;;
background-color: rgb(222, 233, 255);
border-radius: 15px;
}
QPushButton:pressed
{
color: rgb(0,0,0);
font: 10pt &quot;Nunito Medium&quot;;
border-radius: 15px;
background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>File upload</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/image/file (1).png</normaloff>:/image/file (1).png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="folderUpload">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>66</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
color: rgb(0, 0, 0);
font: 10pt &quot;Nunito Medium&quot;;
border : 1px solid rgb(222, 233, 255);
background-color: rgb(255, 255, 255);
}
QPushButton:hover
{
color: rgb(1, 24, 156);
font: 10pt &quot;Nunito Black&quot;;
background-color: rgb(222, 233, 255);
border-radius: 15px;
}
QPushButton:pressed
{
color: rgb(0,0,0);
font: 10pt &quot;Nunito Medium&quot;;
border-radius: 15px;
background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Folder upload</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/image/folder (1).png</normaloff>:/image/folder (1).png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgb(0, 0, 0);
font: 10pt &quot;Nunito Medium&quot;;
background-color: rgb(255, 255, 255);
border: none;
}
QPushButton:hover
{
color: rgb(1, 24, 156);
font: 10pt &quot;Nunito Black&quot;;
background-color: rgb(222, 233, 255);
}
QPushButton:pressed
{
color: rgb(0,0,0);
font: 10pt &quot;Nunito Medium&quot;;
background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/image/D:/download.png</normaloff>:/image/D:/download.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>plusMenu</sender>
   <signal>toggled(bool)</signal>
   <receiver>Menu</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>848</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plusMenu</sender>
   <signal>toggled(bool)</signal>
   <receiver>Menu</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>868</x>
     <y>613</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plusMenu</sender>
   <signal>toggled(bool)</signal>
   <receiver>Menu</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>872</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
